### ISSUE TITLE
Error creating SSL context - Ubuntu20.04

### ISSUE THREAD

**Issue by m-kostrzewa** [Stellarium/stellarium#1265]
### Expected Behaviour

When Getting star catalog update:

```
Error downloading stars4:
Error creating SSL context (error:140A90C4:SSL routines:func(169):reason(196))
```

### Actual Behaviour

Downloads properly

### Steps to reproduce

Press the download button

### System
* Stellarium version: <Name of downloaded installable file?>
* Operating system: Ubuntu 20.04 LTS
* OpenSSL: 1.1.1f

```
apt show openssl
Package: openssl
Version: 1.1.1f-1ubuntu2
```

### Logfile 

Full log: [log.txt](https://github.com/Stellarium/stellarium/files/5215027/log.txt)

Excerpt of the log after pressing the button:

```
[Exoplanets] Updating exoplanets catalog...
[Novae] Updating novae catalog...
qt.network.ssl: QSslSocket: cannot call unresolved function SSLv23_client_method
qt.network.ssl: QSslSocket: cannot call unresolved function SSL_library_init
qt.network.ssl: QSslSocket: cannot call unresolved function SSLv23_client_method
qt.network.ssl: QSslSocket: cannot call unresolved function SSL_library_init
[Exoplanets] Download error: While trying to access "https://stellarium.org/json/novae.json" the following error occured: "Error creating SSL context (error:140A90C4:SSL routines:func(169):reason(196))"
[Novae] Download error: While trying to access "https://stellarium.org/json/novae.json" the following error occured: "Error creating SSL context (error:140A90C4:SSL routines:func(169):reason(196))"
Downloading file "https://github.com/Stellarium/stellarium-data/releases/download/stars-2.0/stars_4_1v0_2.cat"
qt.network.ssl: QSslSocket: cannot call unresolved function SSLv23_client_method
qt.network.ssl: QSslSocket: cannot call unresolved function SSL_library_init
Error downloading file QUrl("https://github.com/Stellarium/stellarium-data/releases/download/stars-2.0/stars_4_1v0_2.cat") :  "Error creating SSL context (error:140A90C4:SSL routines:func(169):reason(196))"
[Exoplanets] Already updating...  will not start again current update is complete.
[Novae] already updating...  will not start again current update is complete.
[Exoplanets] Already updating...  will not start again current update is complete.
[Novae] already updating...  will not start again current update is complete.
```

### Comment

This is another one of those SSL-related errors. However, I was under the impression they're fixed, so that's why I'm reporting.

**Comment by alex-w** [comment]
See also #578

**Comment by m-kostrzewa** [comment]
> See also #578

Saw it, but I don't see how the issue is resolved (or not)? Would installing older libssl version help?

**Comment by alex-w** [comment]
Sorry, we discontinue support an AppImage builds since version 0.20.3 due technical problems of AppImage architecture! Please use snap packages instead.

**Comment by Dimushkashere** [comment]
> This is another one of those SSL-related errors. However, I was under the impression they're fixed, so that's why I'm reporting.

Problem has been resolved in Stellarium-1.0-x86_64.AppImage. I have lower Ubuntu version than you. It was on Stellarium-0.19.1-x86_64.AppImage though.
